util/defaultExists.js

```diff
@@ -0,0 +1,17 @@
+const dir = `${require('os').homedir()}/.terminote/`;
+const setDefault = require('../commands/default');
+const fileExists = require('./fileExists');
+const makeFile = require('./makeFile');
+const error = require('./error');
+
+module.exports = (args) => {
+  return new Promise(async (resolve) => {
+    if (!fileExists('default.json') || !fileExists(require(dir + 'default.json'))) {
+      error('You have no default file. Please make one!', false, 0);
```

Comment:
No need to pass 0 if the second condition is false (no exit code needed because the program does not exit)

Reply:
Should prob use console.warn here instead